sim/shaman/elemental/dragonwrath.go

```diff
@@ -39,11 +47,29 @@ func customFulminationHandler(sim *core.Simulation, spell *core.Spell, result *c
 		copySpell = spell.Unit.RegisterSpell(copyConfig)
 	}
 
-	copySpell.ApplyEffects = fulminationFactory(totalDamage)
+	copySpell.ApplyEffects = damageFactory(totalDamage)
 	cata.CastDTRSpell(sim, copySpell, result.Target)
 }
 
-func fulminationFactory(damage float64) func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+func customEarthquakeHandler(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+	copySpell := spell.Unit.GetSpell(spell.WithTag(71086))
+
+	shaman := spell.Unit.Env.Raid.GetPlayerFromUnit(spell.Unit).(shaman.ShamanAgent).GetShaman()
+	baseDamage := shaman.ClassSpellScaling*0.32400000095 + 0.11*spell.SpellPower()
+
+	if copySpell == nil {
+		copyConfig := cata.GetDRTSpellConfig(spell)
+		copyConfig.Cast = core.CastConfig{}
+		copyConfig.Flags |= core.SpellFlagIgnoreResists // Doesnt get double damage reduced by armor
```

Comment:
I might be missing something, but why is the damage reduced by armor twice? The ApplyEffects function calculates the damage again based on the base damage given. Which should be equivalent to the inital cast's way of doing it, as your're not basing of the new damage on the old one, there should be no second armor reduction, it would not simply ignore armor, no?

Reply:
It might not be reduced twice, i saw that the damage of the replicated spells was doing less damage and assumed that. Adding this is fixing it (in the debug logs, it was at the AfterTargetResistance that damage dropped), but you're right the issue has to be something else, i'm gonna look more into it

Reply:
Phys dot are ignoring armor, so as the base spell is implemtented as a dot and the replicated one not, the damage is not the same without this flag